crates/hc_demo_cli/build.rs

```diff
@@ -39,7 +39,7 @@ fn build(cargo_cmd: &std::ffi::OsStr, tgt: &str) {
     cmd.env_remove("CARGO_ENCODED_RUSTFLAGS");
     cmd.env("CARGO_TARGET_DIR", target_dir.clone());
     cmd.env("HC_DEMO_CLI_INCEPTION", "1");
-    cmd.env("RUSTFLAGS", "-C opt-level=z");
+    cmd.env("RUSTFLAGS", "-C opt-level=z --cfg getrandom_backend=\"custom\"");
```

Comment:
The new version of getrandom needs to be told to use a different backend. This is a change in the 0.6 version of Holochain for how we'll have to build WASMs. You'll see this fix was already applied here -> https://github.com/holochain/holochain/blob/develop/crates/test_utils/wasm/build.rs#L86

Again, something that got missed in this code.